### PR TITLE
Update login admin v2

### PR DESCRIPTION
**Pull request by adreysm09**

### CODE REVIEW

File: src/main/resources/swagger/openapi-admin-doc.yml [hunk 1]
```diff
@@ -294,6 +297,80 @@ paths:
...
+
+  /saveSettingLogin:
+    post:
+      security:
+        - basicAuth:
+            - basicAuth
+      tags:
+        - Admin Change Login
+      description: 'Check and save old Login and new login. Check new login by duplicate.
+      Check new login is duplicate in date base. Send letter with change code'
```
Comment (MalyshEvhen) [inline src/main/resources/swagger/openapi-admin-doc.yml:309]:
```suggestion
      description: |
        Check and save old Login and new login. Check new login by duplicate. Check new login is duplicate in date base. Send letter with change code
```

Comment (malyshevhen) [inline src/main/resources/swagger/openapi-admin-doc.yml]:
```suggestion
      description: 'Check and save old Login and new login. Check new login by duplicate.
        Check new login is duplicate in date base. Send letter with change code'
```

Add tab in line 309.

File: src/main/resources/swagger/openapi-admin-doc.yml [hunk 2]
```diff
@@ -294,6 +297,80 @@ paths:
...
+                $ref: '#/components/schemas/ErrorResponse'
+  /changeLogin:
+    put:
+      security:
+        - basicAuth:
+            - basicAuth
+      tags:
+        - Admin Change Login
+      description: 'Check entered code by saved code. Find admin in date base by saved login.
+      Change admin login to new login.'
```
Comment (malyshevhen) [inline src/main/resources/swagger/openapi-admin-doc.yml]:
```suggestion
      description: 'Check entered code by saved code. Find admin in date base by saved login.
        Change admin login to new login.'
```
Add tab in line 345.

File: src/test/java/baza/trainee/controller/admin/AdminChangeLoginApiControllerTest.java [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    @SneakyThrows
+    @Test
+    void changeCurrentLoginToNewLogin() {
+        String code = "123456";
+        mockMvc.perform(put("/api/admin/changeLogin")
+                        .param("code", code))
+                .andDo(print()).andExpect(status().isNoContent());
+    }
+
+    @WithMockUser(roles = {"ADMIN"})
```
Comment (malyshevhen) [inline src/test/java/baza/trainee/controller/admin/AdminChangeLoginApiControllerTest.java]: Youse JWT authentication instead basic auth.

File: src/test/java/baza/trainee/controller/admin/AdminChangeLoginApiControllerTest.java [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    @Autowired
+    private MockMvc mockMvc;
+
+    @MockBean
+    private AdminLoginService adminLoginService;
+
+    @MockBean
+    private RootUserInitializer rootUserInitializer;
+
+    @WithMockUser(roles = {"ADMIN"})
```
Comment (malyshevhen) [inline src/test/java/baza/trainee/controller/admin/AdminChangeLoginApiControllerTest.java]: Youse JWT authentication instead basic auth.

File: src/main/java/baza/trainee/service/impl/AdminLoginServiceImpl.java [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+import java.util.concurrent.TimeUnit;
+
+import static baza.trainee.constants.MailConstants.ACTIVATION_COD;
+
+@Service
+@RequiredArgsConstructor
+public class AdminLoginServiceImpl implements AdminLoginService {
+
+    @Value("${mail.museum.messagetimeout}")
+    private String messageTimeout;
```
Comment (malyshevhen) [inline src/main/java/baza/trainee/service/impl/AdminLoginServiceImpl.java]: You can use long value instead of string.

**Review by Anatolii-Omelchenko — APPROVED** [review]

**Review by malyshevhen — APPROVED** [review]

**Pull request merged**